presto-native-execution/README.md

```diff
@@ -125,22 +125,27 @@ From the Presto repo run the commands below:
 * `mvn clean install -DskipTests -T1C -pl -presto-docs`
 
 Run IntelliJ IDEA:
+
+For Running HiveExternalWorkerQueryRunner,
 * Edit/Create `HiveExternalWorkerQueryRunner` Application Run/Debug Configuration (alter paths accordingly).
   * Main class: `com.facebook.presto.nativeworker.HiveExternalWorkerQueryRunner`.
   * VM options: `-ea -Xmx5G -XX:+ExitOnOutOfMemoryError -Duser.timezone=America/Bahia_Banderas -Dhive.security=legacy`.
   * Working directory: `$MODULE_DIR$`
   * Environment variables: `PRESTO_SERVER=/Users/<user>/git/presto/presto-native-execution/cmake-build-debug/presto_cpp/main/presto_server;DATA_DIR=/Users/<user>/Desktop/data;WORKER_COUNT=0`
   * Use classpath of module: choose `presto-native-execution` module.
-* Edit/Create `TestPrestoNativeGeneralQueriesJSON` Test Run/Debug Configuration (alter paths accordingly).
-  * Class: `com.facebook.presto.nativeworker.TestPrestoNativeGeneralQueriesJSON`
-  * VM Options: `-ea -DPRESTO_SERVER=/Users/<user>/git/presto_cpp/cmake-build-debug/presto_cpp/main/presto_server -DDATA_DIR=/Users/<user>/Desktop/data`
-  * Working directory: `$MODULE_WORKING_DIR$`
-  * On Apple Silicon
-    * Environment Variables: `DYLD_LIBRARY_PATH=/usr/local/lib`
-* Edit/Create `Presto Client` Application Run/Debug Configuration (alter paths accordingly).
-  * Main class: `com.facebook.presto.cli.Presto`
-  * Program arguments: `--catalog hive --schema tpch`
-  * Use classpath of module: choose `presto-cli` module.
+
+For Running IcebergExternalWorkerQueryRunner,
+* Edit/Create `IcebergExternalWorkerQueryRunner` Application Run/Debug Configuration (alter paths accordingly).
+  * Main class: `com.facebook.presto.nativeworker.IcebergExternalWorkerQueryRunner`.
+  * VM options: `-ea -Xmx5G -XX:+ExitOnOutOfMemoryError -Duser.timezone=America/Bahia_Banderas -Dhive.security=legacy`.
+  * Working directory: `$MODULE_DIR$`
+  * Environment variables: `PRESTO_SERVER=/Users/<user>/git/presto/presto-native-execution/cmake-build-debug/presto_cpp/main/presto_server;DATA_DIR=/Users/<user>/Desktop/data;WORKER_COUNT=0`
+    * When `addStorageFormatToPath = false` **(Default)**
+      `$DATA_DIR/iceberg_data/<catalog_type>` Here `catalog_type` could be `HIVE | HADOOP | NESSIE | REST`
+    * When `addStorageFormatToPath = true`
+      `$DATA_DIR/iceberg_data/<file_format>/<catalog_type>` Here `file_format` could be `PARQUET | ORC | AVRO`
+       And `catalog_type` could be `HIVE | HADOOP | NESSIE | REST`
+  * Use classpath of module: choose `presto-native-execution` module.
```

Comment:
Could you please add a note why addStorageFormatToPath is set to false by default?